diff --git a/src/core/SessionLifecycle.lua b/src/core/SessionLifecycle.lua
index 288cf03c..8be5fa19 100644
--- a/src/core/SessionLifecycle.lua
+++ b/src/core/SessionLifecycle.lua
@@ -115,7 +115,9 @@ function SessionLifecycle.endGameSession(game)
 end
 
 function SessionLifecycle.endProcess()
-  for _, fn in ipairs(processShutdowns) do pcall(fn) end
+  for _, fn in ipairs(processShutdowns) do
+    pcall(fn)
+  end
 end
 
 return SessionLifecycle
